likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/ui/widgets/comment/commentcomposer/view/LMFeedCommentComposerView.kt

```diff
@@ -0,0 +1,176 @@
+package com.likeminds.feed.android.core.ui.widgets.comment.commentcomposer.view
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import androidx.constraintlayout.widget.ConstraintLayout
+import androidx.core.content.ContextCompat
+import com.likeminds.feed.android.core.databinding.LmFeedCommentComposerViewBinding
+import com.likeminds.feed.android.core.ui.base.styles.*
+import com.likeminds.feed.android.core.ui.base.views.LMFeedEditText
+import com.likeminds.feed.android.core.ui.widgets.comment.commentcomposer.style.LMFeedCommentComposerStyle
+import com.likeminds.feed.android.core.utils.LMFeedStyleTransformer
+import com.likeminds.feed.android.core.utils.LMFeedViewUtils.hide
+import com.likeminds.feed.android.core.utils.LMFeedViewUtils.show
+import com.likeminds.feed.android.core.utils.listeners.LMFeedOnClickListener
+
+class LMFeedCommentComposerView : ConstraintLayout {
+
+    constructor(context: Context) : super(context) {
+    }
+
+    constructor(context: Context, attrs: AttributeSet?) : super(context, attrs) {
+    }
+
+    constructor(context: Context, attrs: AttributeSet?, defStyle: Int) : super(
+        context,
+        attrs,
+        defStyle
+    ) {
+    }
+
+    private val inflater =
+        (context.getSystemService(Context.LAYOUT_INFLATER_SERVICE) as LayoutInflater)
+
+    private val binding: LmFeedCommentComposerViewBinding =
+        LmFeedCommentComposerViewBinding.inflate(inflater, this, true)
+
+    //todo: confirm this
+    val etComment: LMFeedEditText
```

Comment:
why todo is written?

Reply:
removed in further PRs as I had to discuss with you